doc/whats-new.rst

```diff
@@ -33,6 +33,8 @@ v0.11.0 (unreleased)
 Breaking changes
 ~~~~~~~~~~~~~~~~
 
+- ``Dataset.T`` has been removed as a shortcut for :py:meth:`transpose()`.
```

Comment:
Generally we don't put `()` in the docs - is this our convention or will the docs not link?

Reply:
good catch

Reply:
It's also in the line below! 
I really don't think it matters if it's only cosmetic, though

Reply:
![image](https://user-images.githubusercontent.com/2448579/47536976-0d8fc080-d878-11e8-8f72-2380a264689b.png)